.circleci/config.yml

```diff
@@ -71,13 +73,15 @@ jobs:
       - run:
           name: Package
           command: >-
```

Comment:
`>-` should be replaced with `|`
and the ramp command "collected" into a single line.